code/modules/hydroponics/grown/towercap.dm

```diff
@@ -110,7 +110,7 @@
 	production = 2
 	yield = 5
 	potency = 50
-	growthstages = 2
+	growthstages = 3
```

Comment:
this isn't related to the pr

Reply:
898088d I added an extra growth stage sprite to bamboo to make their growth a bit smoother but I forgot to include the icons for it >_> NOW they're in.